roles/setup_etcd/README.md

```diff
@@ -0,0 +1,155 @@
+Setup ETCDe
+=========
+
+This Ansible Galaxy Role installs and configure ETCD for Patroni cluster setup.
+
+Note: The role by default installs 3.5.6
+
+Not all Distribution or versions are supported on all the operating systems available.
+
+Note: The role does not configure EDB Postgres Advanced Server or PostgreSQL for replication it only installs ETCD across multiple nodes.
+
+The ansible playbook must be executed under an account that has full privileges.
+
+Requirements
+------------
+
+The requirements for this ansible galaxy role are:
+
+  1. Ansible >= 2.9
+  2. `community.general`
+  3. `edb_devops.edb_postgres` -> `setup_repo` - for installing the EPAS/PG
+     repository
+  4. `edb_devops.edb_postgres` -> `install_dbserver` - for installing the EPAS/PG
+     binaries
+  5. Access etcd RPM/download URL mentioned in [default/main] (./defaults/main.yml)
+
+Role Variables
+--------------
+
+When executing the role via ansible there are three required variables:
+
+  * ***os***
+
+  Operating Systems supported are: CentOS7, RHEL7, CentOS8, RHEL8, Debian10 and Ubuntu20
+
+  * ***pg_version***
+
+  Postgres Versions supported are: 10, 11, 12, 13 and 14
+
+  * ***pg_type***
+
+  Database Engine supported are: PG and EPAS
+
+These and other variables can be assigned in the `pre_tasks` definition of the
+section: *How to include the `setup_etcd` role in your Playbook*
+
+The rest of the variables can be configured and are available in the:
+
+  * [roles/setup_etcd/defaults/main.yml](./defaults/main.yml) 
+  * [roles/setup_etcd/vars/EPAS_RedHat.yml](./vars/EPAS_RedHat.yml)
+  * [roles/setup_etcd/vars/PG_RedHat.yml](./vars/PG_RedHat.yml)
+
+Dependencies
+------------
+
+The `setup_etcd` role does not have any dependencies on any other roles.
+
+Example Playbook
+----------------
+
+### Hosts file content
+
+Content of the `inventory.yml` file:
+
+```yaml
+all:
+  children:
+    primary:
+      hosts:
+        primary1:
+          ansible_host: xxx.xxx.xxx.xxx
```

Comment:
I think we should have a dedicated hostvar in order to flag the servers we want to setup etcd on.
This way, we can easily setup etcd on barman server, pgbouncer server, pem server, etc...
We just need to make sure we have at least 3 etcd nodes.